compiler-plugin/src/main/kotlin/arrow/meta/higherkind/HigherKindPlugin.kt

```diff
@@ -45,7 +47,7 @@ private val KtClass.partialTypeParameters: String
       it.nameAsSafeName.identifier
     }
 
-private val KtClass.arity: Int
+val KtClass.arity: Int
```

Comment:
All these functions are unrelated to autofold and should be moved to the KtUtils.